python/.konan

```diff
@@ -0,0 +1,24 @@
+# This file contains some configurations for your konan deployment
+# Read the comments carefully, and replace any string of the format {tempalte-value} with your actual value
+
+# Section related to where to push your generated image to
+# [docker-registry]
+
+# Possible values include: docker.io, quai.io, registry.gitlab.com and konancr.azurecr.io
```

Comment:
you should include gcr and ecr.

Reply:
Support for gcr and ecr will be added in a separate upcoming feature

Reply:
@mtawfik-95 I think he meant container registries on other cloud providers in general, just make it clear that any registry is possible, not just the ones presented.

Reply:
GCR and ECR in particular (maybe other registries as well) will probably need specific support because how they handle their container registry URLs and login mechanisms,